performance-test/README.md

```
@@ -1,178 +1,103 @@
+* This directory contains Performance Test script of below API endpoint categories grouped inside "Thread Groups".
+	01. A00 Auth Token Generation (Setup)
+	02. A01 Create Mock Identities (Setup)
+	03. A02 Create OIDC Client (Setup)
+	04. S01 OTP Authentication (Execution)
+	05. S02 Biometric Authentication (Execution)
+
+* Open source tools used,
+	01. [Java 21](https://www.oracle.com/java/technologies/downloads/#java21)
+	02. [Apache JMeter 5.6.3](https://jmeter.apache.org/download_jmeter.cgi)
+	03. Libraries
+		* [nimbus-jose-jwt](https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)